frontend/packages/ceph-storage-plugin/integration-tests/views/installFlow.view.ts

```diff
@@ -134,15 +142,21 @@ export class InstallCluster {
     await nsTag.click();
     await subscribeButton.click();
     await browser.wait(until.and(crudView.untilNoLoadersPresent));
+    await browser.wait(until.visibilityOf(searchInputOperators));
+    await searchInputOperators.sendKeys(OCS_OP_NAME);
   }
 
   async createStorageCluster() {
     await goToInstalledOperators();
+    await browser.wait(until.visibilityOf(searchInputOperators));
+    await searchInputOperators.sendKeys(OCS_OP_NAME);
     await click(ocsOperator);
+    // Operators page does not directly show tiles so refresh until it shows
+    await refreshIfNotVisible(storageClusterLink, 5);
```

Comment:
Why 5 ?

Reply:
no need for 10

Reply:
Do we really need to refresh the page 5 times?

Reply:
yes!

Reply:
It will do a maximum of 5 times.

Reply:
Please look at the code guys.

Reply:
I am expecting a reason for choosing the magic number 5.
Code we all see and review.
Nvm, not blocking the PR on this comment.
Thank you

Reply:
Well if it doesn't happen in 5 refreshes then how long should we wait? It can be any arbitrary number. Do you have a number in mind better than 5?

Reply:
With good reason to choose 5, I am fine with 5.
Thanks